libreyolo/export/tensorrt.py

```
@@ -338,6 +338,39 @@ def export_tensorrt(
+                    if "backbone" not in (_lyr.name or "") or _lyr.type == trt.LayerType.SHAPE:
+                        continue
+                    try:
+                        _outs = [_lyr.get_output(_j) for _j in range(_lyr.num_outputs)]
+                        if any(o is None or o.dtype not in (trt.float32, trt.float16) for o in _outs):
+                            continue
+                        _lyr.precision = trt.float32
+                        for _j in range(_lyr.num_outputs):
+                            _lyr.set_output_type(_j, trt.float32)
+                        _npin += 1
+                    except Exception:
+                        continue
```

Ruff (0.15.17)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)